src/crypto/store/memory-crypto-store.js

```diff
@@ -270,6 +273,16 @@ export default class MemoryCryptoStore {
         this._inboundGroupSessions[senderCurve25519Key+'/'+sessionId] = sessionData;
     }
 
+    // Device Data
+
+    getEndToEndDeviceData(txn, func) {
+        func(this._deviceData);
+    }
+
+    storeEndToEndDeviceData(deviceData, txn) {
+        this._deviceData = deviceData;
```

Comment:
wondering if there is any point in this at all, given DeviceList maintains its own copy

Reply:
Yeah, it is a bit pointless but this is for the unit tests anyway so it basically just has to obey the interface.